articles/azure-vmware/request-host-quota-azure-vmware-solution.md

```diff
@@ -17,7 +17,7 @@ In this how-to, you'll request host quota/capacity for [Azure VMware Solution](i
 If you have an existing Azure VMware Solution private cloud and want more hosts allocated, you'll follow the same process.
 
 >[!IMPORTANT]
->It can take up to five business days to allocate the hosts, depending on the number requested.
+>It can take up to five business days to allocate the hosts, depending on the number requested.  So request what is needed for provisioning, so you don't need to request a quota increase as often.
```

Comment:
```suggestion
>It can take up to five business days to allocate the hosts, depending on the number requested. Therefore, request what you need for provisioning to avoid the delays associated with making additional quota increase requests.
```